src/transformers/models/opt/modeling_opt.py

```diff
@@ -511,11 +511,6 @@ def __init__(self, config: OPTConfig):
 
         self.embed_positions = OPTLearnedPositionalEmbedding(num_embeddings, config.hidden_size, self.padding_idx)
 
-        if config.word_embed_proj_dim != config.hidden_size:
-            self.project_out = nn.Linear(config.hidden_size, config.word_embed_proj_dim, bias=False)
```

Comment:
Removing a potential weight of the model could be seriously breaking. Are we sure none of the checkpoints have those?